src/components/message-preview-content/messages.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    previewError: {
```

Comment:
Can we update this to be `messagePreviewError`?

Reply:
Done